seqr/views/apis/summary_data_api_tests.py

```diff
@@ -231,14 +236,18 @@ class LocalSummaryDataAPITest(AuthenticationTestCase, SummaryDataAPITest):
     MANAGER_VARIANT_GUID = 'SV0000006_1248367227_r0004_non'
 
 
-def assert_has_expected_calls(self, users, skip_group_call_idxs=None):
+def assert_has_expected_calls(self, users, skip_group_call_idxs=None, has_ws_access_level_call=False):
```

Comment:
this function is only used twice so theres no reused behavior for `mock_get_ws_access_level` at all anymore - it wold be better to take that check out of the helper and explicitly add the correct check for it after calling this helper